pkg/test/framework.go

```diff
@@ -293,6 +293,10 @@ func (f *Framework) setupLocalCommand() (*exec.Cmd, error) {
 	if ok {
 		watchNamespace = ns
 	}
+	// TODO: The default kubebuilder main.go has no support for using WATCH_NAMESPACE
```

Comment:
This will work fine right now though since `WATCH_NAMESPACE=""` is valid for global operators right?

Might be better to move this TODO to `internal/plugins/golang/plugin.go`.

Reply:
@estroz This TODO is specifically to track the support for namespaced-scoped operators. In that case test-framework has no say over the namespace that the test operator watches.

On the question of introducing support for `WATCH_NAMESPACE` via the plugin modification, I think we're leaning towards not modifying the default scaffolds and leaving this as an advanced step where we provide docs and helpers for the users to make that modification themselves.
So there might not be a TODO for the plugins on this matter.

So I'm leaving this here until we can add those docs.

Reply:
My thought was that the content of this comment doesn't pertain to or discuss testing, only to changing how `main.go` is scaffolded/modified which is done in plugins which is not user-facing. Can you add a note on how this pertains to testing if you intend to keep it here?

On `WATCH_NAMESPACE`: yep, was just wondering whether right now `test local` will work for both global SDK and kubebuilder operators without changes to `WATCH_NAMESPACE` handling logic.